src/people/widgetViews/UserTicketsView.spec.tsx

```diff
@@ -293,4 +296,39 @@ describe('User Tickets View', () => {
       expect(displayName).toBeInTheDocument();
     });
   });
+
+  it('test that the correct calls are made when boxes are clicked for created bounties tab', async () => {
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+    const userBounty = { ...mockBounties[0], body: {} } as any;
+    userBounty.body = {
+      ...userBounty.bounty,
+      owner_id: person.owner_pubkey,
+      title: 'test bounty here',
+      description: 'custom ticket for testing'
+    };
+
+    const mockedPersonAssignedBounites = jest
+      .spyOn(mainStore, 'getPersonAssignedBounties')
+      .mockReturnValue(Promise.resolve([userBounty]));
+    act(async () => {
+      render(
+        <MemoryRouter initialEntries={['/p/1234/usertickets']}>
+          <Route path="/p/:personPubkey/usertickets" component={UserTickets} />
+        </MemoryRouter>
+      );
+      await waitFor(() => {
+        const checkBoxLable = screen.queryByText('Assigned');
+        expect(checkBoxLable).toBeInTheDocument();
+
```

Comment:
@okhot please assert that clicking on the checkboxes triggers a getAssignedBounties call